debezium-standalone-quarkus/src/main/resources/META-INF/beans.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't think that's needed with Quarkus, is it?

Reply:
From docs

> The bean archive is synthesized from:
> * the application classes,
> * dependencies that contain a beans.xml descriptor (content is ignored),

Reply:
Yes, but Quarkus lifts many requirements; i've never needed _beans.xml_ so far.

Reply:
Tahts from *Quarkus* docs https://quarkus.io/guides/cdi-reference#bean_discovery
But I'll try to drop it